### PR TITLE
Bug/fix transferable #179

### LINKED ISSUE #179: [Bug]: Transferable is incorrect, leading to incorrect Total Balance in Provider pane
### What happened?

The transferable amount is incorrect, which results in an incorrect Total Balance in the Provider pane.

This, from the PolkadotJS UI:
Screenshot 2024-10-08 at 3 21 38 PM

Versus:
Screenshot 2024-10-08 at 3 22 49 PM

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by shannonwells**
## Goal

The goal of this PR is to use correct values for transferable value for a given account.
It also updates some polkadot api packages and has some npm audit fixes.

Closes #179 

## Discussion
The transferable balance is not `accountData.free`, but rather, `accountData.free - accountData.frozen`.  See [Kusama | Account Balances: Guide](https://guide.kusama.network/docs/learn-account-balances)

## Checklist

- [x] PR Self-Review and Commenting

## How To Test the Changes

1. Clone the pr branch and run the app locally:
```shell
git co bug/fix-transferable-#179
npm install
npm run dev -- --open
```
2. Connect an account that has known locked values - preferably a Provider account that has already some stake for Capacity.  If you don't have one  you can create one through the app.
3. Go to the PolkadotUI for [Frequency Paseo Accounts page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F0.rpc.testnet.amplica.io#/accounts), and look at the balances for that account.
4. Verify that the values that appear in Provider Dashboard for Transferable, Total, and Locked match what shows in the Polkadot UI.

### CODE REVIEW

**Review by wilwade — APPROVED** [review]
Nice improvements!

- Reviewed code
- Reviewed PR description

**Review by mattheworris — APPROVED** [review]
- Read through the code and balances explanation
- Looks correct, but did not test.
🚢 it!

File: src/lib/polkadotApi.ts [hunk 1]
```diff
@@ -36,16 +36,20 @@ export function getToken(chain: ChainProperties) {
 }
 
 export interface AccountBalances {
-  free: bigint;
-  frozen: bigint;
-  reserved: bigint;
```
Comment (enddynayn) [inline src/lib/polkadotApi.ts:41]: 👍

**Review by enddynayn — APPROVED** [review]
👍 great!

**Pull request merged**